docs/source/guides/migration.rst

```diff
@@ -68,47 +68,46 @@ Next, we will transform the service definition module and breakdown each section
 
 .. admonition:: 💡 Migration Task
 
-   Update the service definition module `service.py` from the BentoML 0.13.1 specification to 1.0.0 specification.
+   Update the service definition module ``service.py`` from the BentoML 0.13.1 specification to 1.0.0 specification.
 
 .. tab-set::
 
     .. tab-item:: 0.13.1
 
-        .. code-block:: python
-            :caption: service.py
+       .. code-block:: python
+          :caption: service.py
 
-            import pandas as pd
+          import pandas as pd
 
-            from bentoml import env, artifacts, api, BentoService
-            from bentoml.adapters import DataframeInput
-            from bentoml.frameworks.sklearn import SklearnModelArtifact
+          from bentoml import env, artifacts, api, BentoService
+          from bentoml.adapters import DataframeInput
+          from bentoml.frameworks.sklearn import SklearnModelArtifact
 
-            @env(infer_pip_packages=True)
-            @artifacts([SklearnModelArtifact('model')])
-            class IrisClassifier(BentoService):
-                @api(input=DataframeInput(), batch=True)
-                def predict(self, df: pd.DataFrame):
-                    return self.artifacts.model.predict(df)
+          @env(infer_pip_packages=True)
+          @artifacts([SklearnModelArtifact('model')])
+          class IrisClassifier(BentoService):
+              @api(input=DataframeInput(), batch=True)
+              def predict(self, df: pd.DataFrame):
+                  return self.artifacts.model.predict(df)
 
     .. tab-item:: 1.0.0
 
-        .. code-block:: python
-            :caption: service.py
+       .. code-block:: python
+          :caption: service.py
 
-            import numpy as np
-            import pandas as pd
+          import numpy as np
+          import pandas as pd
 
-            import bentoml
-            from bentoml.io import NumpyNdarray, PandasDataFrame
+          import bentoml
 
-            iris_clf_runner = bentoml.sklearn.get("iris_clf:latest").to_runner()
+          iris_clf_runner = bentoml.sklearn.get("iris_clf:latest").to_runner()
 
-            svc = bentoml.Service("iris_classifier", runners=[iris_clf_runner])
+          svc = bentoml.Service("iris_classifier", runners=[iris_clf_runner])
 
-            @svc.api(input=PandasDataFrame(), output=NumpyNdarray())
-            def predict(input_series: pd.DataFrame) -> np.ndarray:
-                result = iris_clf_runner.predict.run(input_series)
-                return result
+          @svc.api(input=bentoml.io.PandasDataFrame(), output=bentoml.io.NumpyNdarray())
```

Comment:
Why this change? It's not in the actual example.

Reply:
let me change this back